docs/build/smart-contracts/getting-started/hello-world.mdx

```diff
@@ -60,7 +61,7 @@ members = [
 ]
 
 [workspace.dependencies]
-soroban-sdk = "20.3.2"
+soroban-sdk = "22"
```

Comment:
Genuine question, because I'm not sure: Are there "best practices" or common patterns that are recommended somewhere for how "granular" `Cargo.toml` dependencies should be?

I just realized you can do this, and it could save me a lot of headaches to just specify the major version. But, I don't know if the tutorial should "buck against" conventional wisdom just for the sake of convenience, **if** there are common/accepted reasons to explicitly specify the full semantic version.

Reply:
That's a good question! My main reason for making this change was so that it would match the version that is currently being used when a user creates a new project with `stellar contract init`. 

In terms of best practices, I think it depends on the dependency you're including in the Cargo.toml and if you're confident that it is using semantic versioning, and won't introduce breaking changes in minor release updates. For the sdk since we know that it does use semantic versioning, I think that we're safe just using the major version, which gives Cargo the ability to use any 22 version.